README.md

```diff
@@ -2,19 +2,28 @@
 
 [![Build Status](https://travis-ci.org/immersive-web/webxr.svg?branch=master)](https://travis-ci.org/immersive-web/webxr)
 
-WebXR is driving support for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays on the Web.
+The WebXR device API os for accessing virtual reality (VR) and augmented reality (AR) devices, including sensors and head-mounted displays on the Web. 
 
-This is currently the repository for the [WebVR Community Group][3] and the [WebXR Device API Specification][1]. We have a [charter][4] (in progress).
+|    |             Headset Devices             | Handheld Device e.g. Phone |
+|----|:---------------------------------------:|:--------------------------:|
+| VR | VR Devices, previously handled by WebVR | Magic Window Behaviour     |
+| AR | Mixed Reality Headsets                  | Phone AR                   |
 
-The specification has recently undergone a name change, so expect to see multiple references to "WebVR" sprinkled throughout.
+The [WebXR Device API Specification][1] is the repository of the [Immersive Web Working Group][17]. You can read the [Charter Here][18].
 
+## Taking Part
+
+1. Read the [code of conduct][19]
+2. See if your issue is bring discussed in the [issues][8], or if your idea is being discussed in the [proposals repo][20].
```

Comment:
typo: bring

s/is being/is or has been discussed/? Note, though, that the default view is only open issues.